portfolio/src/main/webapp/script.js

```diff
@@ -98,4 +98,37 @@ async function sayHelloUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
   document.getElementById('hello-sell').innerText = quote;
-}
\ No newline at end of file
+}
+
+function getServerStats() {
+  fetch('/data').then(response => response.json()).then((jsonStr) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+    const statsListElement = document.getElementById('hello-sell');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(
+        createListElement('First string: ' + jsonStr[0]));
+    statsListElement.appendChild(
+        createListElement('Second string: ' + jsonStr[1]));
+    statsListElement.appendChild(
+        createListElement('Third string: ' + jsonStr[2]));
+  });
+}
+
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
+
+async function formLoad() {
+  fetch('/form').then(response => response.json()).then((jsonStr) => {
```

Comment:
Check out the documentation for [`await`](http://go/mdn/JavaScript/Reference/Operators/await). Basically, you can `await` anything which returns a `Promise<T>` and it will give you the `T` directly. For example, `fetch` returns a `Promise<Response>`, and the `Response` is what it passes to your callback in `then(response => response.json())`. If you do `const response = await fetch('/form');`, the `response` variable is the `Response` value directly.